pkgs/applications/window-managers/leftwm/default.nix

```diff
@@ -20,7 +21,22 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-wn5DurPWFwSUtc5naEL4lBSQpKWTJkugpN9mKx+Ed2Y=";
   };
 
-  cargoHash = "sha256-TylRxdpAVuGtZ3Lm8je6FZ0JUwetBi6mOGRoT2M3Jyk=";
+  cargoPatches = [
+    # This pacth can be removed with the next version bump, it just updates the `time` crate
+    (fetchpatch {
+      url = "https://github.com/VuiMuich/leftwm/commit/9358bebb11df19f46d0813723959518498d812b2.patch";
```

Comment:
Does the [upstream fix](https://github.com/leftwm/leftwm/commit/7e7fcab2c141bac303b70b33a5890ec8b0666f74) not apply?

If not:

If the branch this commit is on is ever rebased or deleted, GitHub will GC this commit and the URL will stop working, so it's better to include the patch file in Nixpkgs for fixes that aren't upstream.

Reply:
In upstream the `time` crate is updated over a bunch of steps so the patch to fix the 1.80.0-issue fails to apply from the last release. The guys at Void Linux use `cargo update --package time@0.3.30 --precise 0.3.35` ([see](https://github.com/void-linux/void-packages/pull/51615)) and I thought the quickest way to do this here is apply it to a separate branch on my own fork. This all is obsolete with the next release, but this might be away a couple of weeks and we will look definitely to get the new version in for `24.11`.

Reply:
It shouldn't be any quicker than including a diff file, and once it's committed, it's in the Nixpkgs history forever, and that's relevant if one day somebody has to e.g. do a bisect, or figure out how something worked in the past and then they find that now the URL is broken.